### PR TITLE
Added settings for BrowserSync.

### PR DESCRIPTION
**Pull request by puredazzle**
Added default settings for BrowserSync and only hash files in production (this is required for BrowserSync to work).

### CODE REVIEW

**Comment by vinkla** [comment]
Do we still need all `npm scripts` or could we remove any of them?

File: webpack.mix.js [hunk 1]
```diff
@@ -12,12 +12,26 @@ const mix = require('laravel-mix').mix;
...
+    proxy: 'wordplate.dev',
+    files: [
+        `${themePath}/**/*.php`,
+        `${assetsPath}/**/*.js`,
+        `${assetsPath}/**/*.css`
+    ]
+});
+
+mix.js(`${resources}/scripts/app.js`, `${assetsPath}/scripts`)
+    .sass(`${resources}/styles/app.scss`, `${assetsPath}/styles`, {
```
Comment (vinkla) [inline webpack.mix.js]: Maybe write this as `mix.sass` instead?

File: webpack.mix.js [hunk 2]
```diff
@@ -12,12 +12,26 @@ const mix = require('laravel-mix').mix;
  */
 
 const resources = 'resources/assets';
```
Comment (vinkla) [inline webpack.mix.js]: @puredazzle what do you think about adding a `theme` variable? Will make it easier to change the name.

**Comment by puredazzle** [comment]
@vinkla regarding the NPM scripts: https://github.com/wordplate/wordplate/pull/164

File: webpack.mix.js (hunk 1 above)
Comment (puredazzle) [inline webpack.mix.js]: Sounds good!

File: webpack.mix.js (hunk 2 above)
Comment (puredazzle) [inline webpack.mix.js]: Wrong line?

Comment (puredazzle) [inline webpack.mix.js]: But I don't think we need theme as a variable as it is only one place you need to update it. The proxy don't always reflect the theme name.

Comment (vinkla) [inline webpack.mix.js]: Yeah, true.

**Comment by vinkla** [comment]
Thanks 👍

**Pull request merged**